### PR TITLE
Fixed merge conflicts and removed unit test

### PR DESCRIPTION
**Pull request by jainammshah12**
I made some errors in the back-end which caused a couple of bugs. Also I will rewrite my unit test and make one more pull request.

### CODE REVIEW

File: server/real_estate/src/test/java/com/realestate/real_estate/repos/PropertyRepositoryTests.java [hunk 1]
```diff
@@ -26,6 +26,7 @@ public class PropertyRepositoryTests {
     private PropertyRepository propertyRepository;
 
     // Arrange
+    String id = new String();
```
Comment (huy2272) [inline server/real_estate/src/test/java/com/realestate/real_estate/repos/PropertyRepositoryTests.java:29]: Why do you need this?

**Review by huy2272 — APPROVED** [review]
Approved but left a question

File: server/real_estate/src/test/java/com/realestate/real_estate/repos/PropertyRepositoryTests.java (hunk 1 above)
Comment (jainammshah12) [reply server/real_estate/src/test/java/com/realestate/real_estate/repos/PropertyRepositoryTests.java:29]: Honestly, I really don't know. It was a part of the unit test, I just changed it to String datatype. Should I remove it?

Comment (huy2272) [reply server/real_estate/src/test/java/com/realestate/real_estate/repos/PropertyRepositoryTests.java:29]: Nah, if everything is working let's not touch it

**Pull request merged**